### PR TITLE
Extended setup

### PR DESCRIPTION
**Pull request by JollyBrackets**

### CODE REVIEW

File: yarn-error.log [hunk 1]
```diff
@@ -0,0 +1,32 @@
+Arguments:
```
Comment (piggydoughnut) [inline yarn-error.log]: this should be in .gitignore 

now we all know that you use Windows :)

Comment (nicksarafa) [reply yarn-error.log]: lmao windows is for squares jk lol

File: public/src/App.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.btn__content {
+  text-transform: capitalize !important;
```
Comment (nicksarafa) [inline public/src/App.css]: `!important` ???? who uses important

Comment (roma219) [reply public/src/App.css]: @nicksarafa Why not in the case of overriding vuetify built-in styles?

**Comment by nicksarafa** [comment]
👍

File: public/src/App.css (hunk 2 above)
Comment (nicksarafa) [reply public/src/App.css]: 👍

**Pull request merged**